cookbook/frontend/bower.rst

```diff
@@ -0,0 +1,125 @@
+.. index::
+    single: Front-end; Bower
+
+Using Bower with Symfony
+========================
+
+Symfony and all its packages are perfectly managed by Composer. Bower is a
+dependency management tool for front-end dependencies, like Twitter Bootstrap
+or jQuery. Since Symfony isn't purely a back-end framework, it can't deliver
+you much help with Bower. Fortunately, it is very easy to use!
+
+Installing Bower
+----------------
+
+Bower_ is build on top of NodeJS_. Make sure you have that installed and
+then run:
+
+.. code-block:: bash
+
+    $ npm install -g bower
+
+After this command succeeded, run ``bower`` in your terminal to find out if
+it's installed correctly.
+
+.. tip::
+
+    If you don't want to have NodeJS on your computer, you can also BowerPHP_
+    for an unofficial PHP port. Beware that this is still in an alpha state. If
+    you're using BowerPHP, use ``bowerphp`` instead of ``bower`` in the
+    examples.
+
+Configuring Bower in your Project
+---------------------------------
+
+By default, Bower will install all front-end dependencies in a
+``bower_packages/`` in the root of your project. In Symfony, only the ``web/``
+directory has public access, so you have to configure Bower to install it in
+the ``web/`` directory instead.
+
+A common practice is to put your front-end dependencies in
+``web/assets/vendor/``. You can configure this directory in a ``.bowerrc`` file
```

Comment:
I like `web/assets/vendor/` (I think I stole this a few years back from Ryan's front-end talk). Afaik, Assetic is just putting them in `web/`